mobile-sso/swift-demo/test/ViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ViewController.swift
+//  test
+//
+//  Created by Derrick Lin on 9/1/20.
+//  Copyright © 2020 Derrick Lin. All rights reserved.
+//
+
+import UIKit
+import WebKit
+import Alamofire
+import SwiftyJSON
+
+
+class ViewController: UIViewController, WKNavigationDelegate, WKUIDelegate {
+
+    @IBOutlet var mywebview: WKWebView!
+    
+    func renderSite(payload: String) {
+        var components = URLComponents()
+        components.scheme = "http"
+        components.host = "localhost"
+        components.port = 5000
+//        components.scheme = "https"
+//        components.host = "sleepy-shelf-33354.herokuapp.com"
+//
+        components.queryItems = [
+            URLQueryItem(name: "title", value: "Hovsep"),
+            URLQueryItem(name: "identifier", value: "the_hovsep_identifier"),
+            URLQueryItem(name: "payload", value: payload)
+        ]
+        let url = components.url!
+        mywebview.load(URLRequest(url: url))
+    }
+    
+    func login() {
+        let parameters = [
+            "email": "dmatt+9898@disqus.com",
+            "password": "pwd"
+        ]
+//        let parameters = [
```

Comment:
this seems like it needs to be cleaned up

Reply:
intended